node/utils/address.ts

```diff
@@ -1,8 +1,6 @@
 import { AddressType, DELIVERY, PICKUP_IN_POINT } from '../constants'
 
-export function getSelectedDeliveryAddress(
-  selectedAddresses: CheckoutAddress[]
-) {
+export function getSelectedAddress(selectedAddresses: CheckoutAddress[]) {
```

Comment:
Should we worry that this exists?
https://github.com/vtex-apps/connector-paypal-commerce-platform/blob/3e8f83fdd54a5e865ffa67f3c5f0d36b5c9d6eb5/node/utils/shipping.ts#L32-L40

Reply:
it doesn't seem to be used anywhere inside that app, but we could talk with that team and ask them to remove this code